sale_elaboration/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Sale Elaboration",
     "summary": "Set an elaboration for any sale line",
-    "version": "15.0.1.0.1",
+    "version": "15.0.2.0.1",
```

Comment:
It should be (and so the migration folder):

```suggestion
    "version": "15.0.2.0.0",
```